pkg/route/api/defaults.go

```diff
@@ -0,0 +1,24 @@
+
+package api
+
+import (
+	"k8s.io/kubernetes/pkg/runtime"
+)
+
+func addDefaultingFuncs(scheme *runtime.Scheme) {
```

Comment:
You don't need to define defaults on the internal type - they are only applied to the external types (in this case v1).